Hymns/Common/SearchBar.swift

```diff
@@ -45,6 +47,10 @@ struct SearchBar: UIViewRepresentable {
 
     func updateUIView(_ uiView: UISearchBar, context: UIViewRepresentableContext<SearchBar>) {
         uiView.text = text
+        if isFirstResponder && !context.coordinator.didBecomeFirstResponder {
```

Comment:
What does first responder do?

Reply:
Instead of passing this in, could we just make an extension of it in the search view? That way we don't need to pass this in from the home screen

Reply:
First responder is just a bool that we can set, whenever it is true then the search bar will be the first thing selected in a view. 

We still don't need to pass it in since it has a default initialization value of false. It is optional to pass it in as an argument. That is why I passed it in from searchview but not from home screen. Is that what you mean?

Reply:
ahh, I see. okay, sounds good then! I didn't see the default value :)